metrics/src/main/java/com/baeldung/metrics/aspectj/ObjectRunner.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.metrics.aspectj;
+
+import com.codahale.metrics.annotation.Timed;
+import io.astefanutti.metrics.aspectj.Metrics;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Metrics( registry = ObjectRunner.REGISTRY_NAME)
+public class ObjectRunner {
+
+    private static final Logger logger = LoggerFactory.getLogger(ObjectRunner.class);
+    public static final String REGISTRY_NAME = "ObjectRunner";
+
+    @Timed(name = "timerName")
+    public void run() {
+        logger.info("run");
+        try {
+            Thread.sleep(1000L);
+        } catch (InterruptedException e)
+        {
```

Comment:
This `{` belongs on the previous line
It's conventional to comment an empty catch block too

```
} catch (InterruptedException e) {
   // ignore this exception
}
```